flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/CDU/A320_Neo_CDU_FlightPlanPage.js

```diff
@@ -341,14 +342,18 @@ class CDUFlightPlanPage {
                     const lastRouteIndex = fpm.getLastIndexBeforeApproach();
                     //const departureWp = firstRouteIndex > 1 && fpm.getDepartureWaypoints().indexOf(wp) !== -1;
 
-                    if (mcdu.flightPlanManager.getOriginTransitionAltitude() >= 100 && wp.legAltitude1 > mcdu.flightPlanManager.getOriginTransitionAltitude()) {
-                        altitudeConstraint = (wp.legAltitude1 / 100).toFixed(0).toString();
-                        altitudeConstraint = `FL${altitudeConstraint.padStart(3,"0")}`;
-                    } else {
-                        altitudeConstraint = wp.legAltitude1.toFixed(0).toString().padStart(5,"\xa0");
+                    // TODO all this constraint code is very sussy
+
+                    if (hasAltConstraint) {
+                        if (mcdu.flightPlanManager.getOriginTransitionAltitude() >= 100 && wp.legAltitude1 > mcdu.flightPlanManager.getOriginTransitionAltitude()) {
+                            altitudeConstraint = (wp.legAltitude1 / 100).toFixed(0).toString();
+                            altitudeConstraint = `FL${altitudeConstraint.padStart(3,"0")}`;
+                        } else {
+                            altitudeConstraint = wp.legAltitude1.toFixed(0).toString().padStart(5,"\xa0");
+                        }
```

Comment:
Just some formatting suggestions for the sake of consistency. I know this is how it was previously, but it's gonna bother me if I see it like this later 😄.

```suggestion
                    if (hasAltConstraint) {
                        if (mcdu.flightPlanManager.getOriginTransitionAltitude() >= 100 && wp.legAltitude1 > mcdu.flightPlanManager.getOriginTransitionAltitude()) {
                            altitudeConstraint = (wp.legAltitude1 / 100).toFixed(0).toString();
                            altitudeConstraint = `FL${altitudeConstraint.padStart(3, "0")}`;
                        } else {
                            altitudeConstraint = wp.legAltitude1.toFixed(0).toString().padStart(5, "\xa0");
                        }
```

Reply:
I think we will replace this code entirely in cfms-20, but can do this now. :D